src/runtime/tests/particle-interface-loading-with-slots-test.ts

```diff
@@ -61,119 +60,51 @@ describe('particle interface loading with slots', () => {
 
     // Add one more element.
     await inStore.add(Entity.identify(new inStore.entityClass({value: 'foo3'}), 'subid-3'));
-    // TODO(sjmiles): uses old render data, will be repaired in subsequent PR
-    // slotComposer
-    //   .newExpectations()
-    //   .expectRenderSlot('SingleSlotParticle', 'annotation', {contentTypes: ['model']})
-    //   .expectRenderSlot('MultiplexSlotsParticle', 'annotationsSet', {contentTypes: ['model']})
-    //   .expectRenderSlot('MultiplexSlotsParticle', 'annotationsSet', {contentTypes: ['model'], times: 2, isOptional: true});
+    observer
+       .newExpectations()
+       .expectRenderSlot('SingleSlotParticle', 'annotation')
+       ;
     await arc.pec.idle;
-    //await slotComposer.expectationsCompleted();
-
-    // TODO(sjmiles): render data no longer captured by slot objects
-    //verifyFooItems(slot, {'subid-1': 'foo1', 'subid-2': 'foo2', 'subid-3': 'foo3'});
+    await observer.expectationsCompleted();
   });
 
   it('multiplex recipe with slots - init context later', async () => {
     // This test is different from the one above because it initializes the transformation particle context
     // after the hosted particles are also instantiated.
     // This verifies a different start-render call in slot-composer.
-    const {manifest, recipe, slotComposer, arc} = await initializeManifestAndArc();
+    const {manifest, recipe, observer, arc} = await initializeManifestAndArc();
 
-    // TODO(sjmiles): contexts deprecated
-    //(slotComposer.getAvailableContexts()[0] as ProvidedSlotContext).container = null;
     const inStore = await instantiateRecipeAndStore(arc, recipe, manifest);
 
-    // Wait for the hosted slots to be initialized in slot-composer.
-    await new Promise((resolve, reject) => {
-      const myInterval = setInterval(() => {
-        if (slotComposer.consumers.length === 3) { // last 2 are hosted slots
-          resolve();
-          clearInterval(myInterval);
-        }
-      }, 10);
-    });
-
-    // TODO(sjmiles): uses old render data, will be repaired in subsequent PR
-    // slotComposer
-    //   .newExpectations()
-    //   .expectRenderSlot('SingleSlotParticle', 'annotation', {contentTypes: ['template', 'model'], times: 2})
-    //   .expectRenderSlot('MultiplexSlotsParticle', 'annotationsSet', {contentTypes: ['template', 'model']})
-    //   .expectRenderSlot('MultiplexSlotsParticle', 'annotationsSet', {contentTypes: ['model'], times: 2, isOptional: true});
-
-    // TODO(sjmiles): contexts/containers/consumers deprecated
-    // tslint:disable-next-line: no-any
-    //(slotComposer.getAvailableContexts()[0] as ProvidedSlotContext).container = {'subid-1': 'dummy-container1', 'subid-2': 'dummy-container2', 'subid-3': 'dummy-container3'} as any;
-    //slotComposer.consumers[0].onContainerUpdate({}, undefined);
-
+    observer
+      .newExpectations()
+      .expectRenderSlot('SingleSlotParticle', 'annotation', {times: 2})
+      ;
```

Comment:
(very embarrassing nit) indentation should be 4

Reply:
Yes, you are right. Multi-indenting bugs me aesthetically, but I have to let that go.